.travis.yml

```diff
@@ -1,8 +1,14 @@
 language: bash
+sudo: required
```

Comment:
```
+ sudo: required
```

Since there’s no command in the current .travis.yml that uses sudo, it’s not clear why sudo’s being required here?

Reply:
I think it was a copy and paste from somewhere else. I'll try removing it and see what happens.